README.md

```diff
@@ -1,2 +1,24 @@
 # node-backend
-Node api for dead crumb trail
+Node rest api for dead-crumb-trail webapp 
+
+## Setup
+Once you clone the repo, follow the below process to run/test/build:
+
+1) Pull Dependencies
+    npm install
+
+2) Create database
+    - Install mongo https://docs.mongodb.com/guides/server/install/
+    - Create a db named starter https://www.mongodb.com/basics/create-database
+
+2) Running locally
+    npm run dev
+
+3) Running unit test cases
+    npm run test:unit
+
+4) Runiing end to end test case locally
+    npm run test:e2e:local
+    
+5) Running end to end test case on docker container
+    npm run test:e2e
```

Comment:
These commands are pretty obvious from the `package.json` itself. Do we need to repeat it in the readme again? Also, e2e tests need an environment file to be configured with a separate db name to ensure you have an isolated place with no data to start running your tests.

Reply:
@Prabhat-Singh-Pawar Also, once you've made changes for the comments, change the branch name to `feature/dct-21-create-readme-for-the-api` and open a new PR. That way the linear task will start tracking the PR.